index/src/transitive.rs

```diff
@@ -0,0 +1,19 @@
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+use cargo_tally::Crate;
+
+#[derive(Serialize, Deserialize)]
+pub struct TranitiveCrateDeps {
+    pub name: String,
+    /// Crates that depend on this crate
+    depended_on: Vec<Someting>,
```

Comment:
Do we want to use a `Vec<Something>` or just total each crates transitive depended on count.